tests/lib/assert_helper.js

```diff
@@ -244,15 +244,23 @@ PresenceMessage.prototype.assert_get_v2_response = function(message) {
     } else {
       value[client.userId] = userHash = { clients: {} };
     }
+    
     userHash.clients[client.clientId] = client.userData;
     userHash.userType = client.userType;
+
+    if (data) {
+      userHash.data = data;
+    };
+
   });
 
-  assert.deepEqual({
+  var expectedMessage = {
     op: 'get',
     to: this.scope,
     value: value
-  }, message, JSON.stringify(value)+' vs '+JSON.stringify(message.value));
+  }
+
+  assert.deepEqual(expectedMessage, message, JSON.stringify(expectedMessage)+' vs '+JSON.stringify(message));
 };
```

Comment:
It's an ancient custom, but make sure the lines break ~ 80 chars.  So, break this line after *message*?